src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -79,57 +70,6 @@ class RecordingIngestMessage:
     payload_with_headers: bytes
 
 
-@metrics.wraps("replays.usecases.ingest.ingest_recording_chunked")
-def ingest_recording_chunked(
-    message_dict: RecordingSegmentMessage, transaction: Span, current_hub: Hub
-) -> None:
-    """Ingest chunked recording messages."""
-    with current_hub:
-        with transaction.start_child(
-            op="replays.usecases.ingest.ingest_recording_chunked",
-            description="ingest_recording_chunked",
-        ):
-            cache_prefix = replay_recording_segment_cache_id(
-                project_id=message_dict["project_id"],
-                replay_id=message_dict["replay_id"],
-                segment_id=message_dict["replay_recording"]["id"],
-            )
-            parts = RecordingSegmentParts(
-                prefix=cache_prefix, num_parts=message_dict["replay_recording"]["chunks"]
-            )
-
-            try:
-                recording_segment_with_headers = collate_segment_chunks(parts)
-            except ValueError:
-                logger.exception("Missing recording-segment.")
-                return None
-
-            logger.info(
-                "ingest_recording_chunked.info",
-                extra={
-                    "organization_id": message_dict["org_id"],
-                    "project_id": message_dict["project_id"],
-                    "replay_id": message_dict["replay_id"],
-                    "num_parts": message_dict["replay_recording"]["chunks"],
-                    "size_compressed": len(recording_segment_with_headers),
-                },
-            )
-            message = RecordingIngestMessage(
-                replay_id=message_dict["replay_id"],
-                key_id=message_dict.get("key_id"),
-                org_id=message_dict["org_id"],
-                project_id=message_dict["project_id"],
-                received=message_dict["received"],
-                retention_days=message_dict["retention_days"],
-                payload_with_headers=recording_segment_with_headers,
-            )
-            ingest_recording(message, transaction)
-
-            # Segment chunks are always deleted if ingest behavior runs without error.
-            with metrics.timer("replays.process_recording.store_recording.drop_segments"):
-                parts.drop()
-
-
 @metrics.wraps("replays.usecases.ingest.ingest_recording_not_chunked")
 def ingest_recording_not_chunked(
```

Comment:
This should be renamed to `ingest_recording`.  The other `ingest_recording` function can be renamed to `_ingest_recording` or merged into the body of this function.